commands/user.go

```diff
@@ -372,6 +399,51 @@ func updateUserEmailCmdF(c client.Client, cmd *cobra.Command, args []string) err
 	return nil
 }
 
+func changePasswordUserCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	password, _ := cmd.Flags().GetString("password")
+	current, _ := cmd.Flags().GetString("current")
+	hashed, _ := cmd.Flags().GetBool("hashed")
+
+	if password == "" {
+		var confirm string
+		fmt.Printf("Are you changing your own password? (YES/NO): ")
+		fmt.Scanln(&confirm)
+		if confirm == "YES" {
+			fmt.Printf("Current password: ")
+			var err error
+			current, err = getPasswordFromStdin()
+			if err != nil {
+				return errors.Wrap(err, "couldn't read password")
+			}
+		}
+
+		fmt.Printf("New password: ")
+		var err error
+		password, err = getPasswordFromStdin()
+		if err != nil {
+			return errors.Wrap(err, "couldn't read password")
+		}
+	}
+
+	user := getUserFromUserArg(c, args[0])
+	if user == nil {
+		return errors.New("couldn't find user '" + args[0] + "'")
+	}
+
+	if hashed {
+		if _, resp := c.UpdateUserHashedPassword(user.Id, password); resp.Error != nil {
+			return errors.Wrap(resp.Error, "changing user password failed")
```

Comment:
I'd change this to `changing hashed user password failed` so we can differentiate it from the other error